### DISCUSSION TITLE
Set default view of the Calendar

### DISCUSSION THREAD

**Discussion by kaowutong** [kimai/kimai#3936] [General]
Hello,
I use the calendar to schedule the weekly activity of my team members.
When I access the calendar, the default view is the month view.
I would like to set the week view by default.
Is this possible?
Can you tell me what to do?

Thanks in advance.

**Comment by kevinpapst** [comment]
User preferences:
<img width="466" alt="Bildschirm­foto 2023-03-19 um 00 28 04" src="https://user-images.githubusercontent.com/533162/226145252-4afb7b87-485a-4e22-9144-d42bf676a57b.png">

**Reply by kaowutong** [reply]
thank you
